From vapier@gentoo.org  Tue Dec 22 12:00:41 2009
From: Mike Frysinger <vapier@gentoo.org>
Date: Fri, 18 Dec 2009 16:33:01 -0500
Subject: USB: cypress_m8: stop using USB debug driver config
To: Lonnie Mendez <dignome@gmail.com>
Cc: linux-usb@vger.kernel.org
Message-ID: <1261171983-20131-1-git-send-email-vapier@gentoo.org>


The USB_SERIAL_DEBUG Kconfig is for the USB serial debug driver, not for
generically enabling debug output in random USB serial drivers.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/cypress_m8.c |    6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

--- a/drivers/usb/serial/cypress_m8.c
+++ b/drivers/usb/serial/cypress_m8.c
@@ -70,11 +70,7 @@
 #include "cypress_m8.h"
 
 
-#ifdef CONFIG_USB_SERIAL_DEBUG
-	static int debug = 1;
-#else
-	static int debug;
-#endif
+static int debug;
 static int stats;
 static int interval;
 
